compatibility/bazel_tools/create-daml-app/testDeps.json

```diff
@@ -5,6 +5,8 @@
     "@types/puppeteer": "^3.0.1",
     "@types/wait-on": "^4.0.0",
     "puppeteer": "^4.0.1",
-    "wait-on": "^4.0.2"
+    "wait-on": "^4.0.2",
+    "@types/babel__traverse": "7.18.1"
```

Comment:
Are you deliberately downgrading or upgrading? I think it's worth leaving a comment. If upgrading, I think ^7.18.1 would be fine here.

nitpick: I would add this dependency to the top.

Reply:
I am deliberately pinning to a specific version because `7.18.1` works but `7.18.3` doesn't. I'd love to comment but JSON doesn't support comments AFAIK.

Reply:
Also see #14610 (and further linked PRs from there) for a bit more context on this.